src/components/forms/forms.js

```diff
@@ -0,0 +1,404 @@
+import { useState } from 'react';
+import { h, div, label, input, span, button } from 'react-hyperscript-helpers';
+import { cloneDeep } from 'lodash/fp';
+import { SearchSelectOrText } from '../SearchSelectOrText';
+import { Theme } from '../../libs/theme';
+import './forms.css';
+
+export const FormFieldTypes = {
+  SELECT: 'select',
+  MULTITEXT: 'multitext',
+  CHECKBOX: 'checkbox',
+  SLIDER: 'slider',
+  TEXT: 'text',
+  NUMBER: 'number'
+};
+
+export const styles = {
+  inputStyle: {
+    padding: '25px 15px',
+    width: '100%'
+  },
+  flexRow: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    width: '100%'
+  }
+};
+
+//---------------------------------------------
+// Form Behavior
+//---------------------------------------------
+const validateRequired = (value) => {
+  return value === undefined || value === null || value === ''
+    ? 'Please enter a value'
+    : null;
+};
+
+const extractValidators = (config) => {
+  const { required, validator } = config;
+  const ret = [];
+
+  if (required) {
+    ret.push(validateRequired);
+  }
```

Comment:
created this function bc long term, it might be useful to add different types of validators, like Array.minLength